client/src/Components/Dialog/DataSelectDialog.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {
+  List,
+  ListItem,
+  ListItemText,
+  DialogTitle,
+  Dialog,
+  Divider
+} from "@material-ui/core";
+
+const DataSelectDialog = props => {
```

Comment:
This component is what I was looking for! Good job, this is essentially a presentational component